student/models.py

```diff
@@ -8,6 +8,21 @@ class EducationalInstitution(models.Model):
     def __str__(self):
         return self.name
 
+    """
+    Gets all the institutions from the DB, ordered by their name
+    """
+
+    @classmethod
+    def get_institutions(cls):
+        return cls.objects.order_by("name")
+
+    """
+    Gets all the students that are enrolled at the called on institution
+    """
+
+    def get_institution_students(self):
```

Comment:
this does not seem to provide any additional logical functionality beyond what Django already provides - unless you have a specific application use for this list, in which case the method name should reflect that.

Reply:
Recruiters can use this functions to look at students only from specific institutions, maybe it's better to create a similar functions inside the Student model and then filter the students in order to achieve this? This way we could also get a list of institutions instead of only one.